### PR TITLE
Add Timeline synchronization command

### PR DESCRIPTION
**Pull request by Remg**
1. Command `app:algolia:synchronize` now updates Algolia index settings **if defined**.
2. Introduced the command `app:timeline:synchronize` which synchronizes the entities and index settings related to the Timeline with Algolia.

### CODE REVIEW

File: src/Command/AlgoliaSynchronizeCommand.php [hunk 1]
```diff
@@ -2,75 +2,77 @@
...
+        foreach ($this->getEntitiesToIndex($input) as $entity) {
             $output->write('Synchronizing entity '.$entity.' ... ');
-            $nbIndexes = $this->synchronizeEntity($entity);
+            $nbIndexes = $this->algolia->reIndex($entity);
             $output->writeln('done, '.$nbIndexes.' records indexed');
         }
     }
 
-    private function synchronizeEntity($className)
+    protected function getEntitiesToIndex(InputInterface $input): array
```
Comment (HeahDude) [inline src/Command/AlgoliaSynchronizeCommand.php]: You should pass the entity name directly.

File: tests/Command/TimelineSynchronizeCommandTest.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+/**
+ * @group functional
+ */
+class TimelineSynchronizeCommandTest extends SqliteWebTestCase
+{
+    public function testCommand()
+    {
+        $output = $this->runCommand('app:timeline:synchronize');
+
+        $this->assertContains('Synchronizing entity AppBundle\Entity\Timeline\Profile ... done, 5 records indexed', $output);
```
Comment (thomasbisignani) [inline tests/Command/TimelineSynchronizeCommandTest.php]: Can you write in multiple lines here ?

File: tests/Command/TimelineSynchronizeCommandTest.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        $this->assertContains('Synchronizing entity AppBundle\Entity\Timeline\Theme ... done, 5 records indexed', $output);
```
Comment (thomasbisignani) [inline tests/Command/TimelineSynchronizeCommandTest.php]: Same here and the next line

File: tests/Test/Algolia/DummyIndexer.php [hunk 4]
```diff
@@ -49,4 +50,13 @@ public function reset(): void
         $this->entitiesToIndex = [];
         $this->entitiesToUnIndex = [];
     }
+
+    public function getManualIndexer(EntityManager $em)
```
Comment (thomasbisignani) [inline tests/Test/Algolia/DummyIndexer.php]: missing return type

File: tests/Command/TimelineSynchronizeCommandTest.php (hunk 2 above)
Comment (Remg) [reply tests/Command/TimelineSynchronizeCommandTest.php]: denied

**Comment by thomasbisignani** [comment]
:ok_hand:

**Pull request merged**